projects/kit/components/calendar-range/calendar-range.template.html

```diff
@@ -29,7 +29,7 @@
             class="t-menu"
         >
             <button
-                *ngFor="let item of items | tuiMapper: mapper : min : max : minLength : (otherDateText$ | async)"
+                *ngFor="let item of items | tuiMapper: mapper : min : max : minLength : (otherDateText$ | async) || ''"
```

Comment:
Let's add `()` so it's clear at a glance what this fallback is applied to

Reply:
Prettier removes it, looks like its obvious for it 😂.

I remove the fallback and use default args in the template instead.